docs/content/guides/author-apps/azure/howto-azure-container-instances/index.md

```diff
@@ -0,0 +1,201 @@
+---
+type: docs
+title: "How-To: Deploy an Application to Azure Container Instances"
+linkTitle: "Deploy to ACI"
+description: "Learn how to configure and deploy an application to Azure Container Instances"
+weight: 500
+slug: 'azure-container-instances'
+categories: "How-To"
+tags: ["Azure","containers"]
+---
+
+This how-to guide will provide an overview of how to:
+
+- Configure and deploy a Radius [Environment]({{< ref "/guides/deploy-apps/environments/overview" >}}) that uses [Azure Container Instances (ACI)](https://learn.microsoft.com/en-us/azure/container-instances/) as the compute provider.
+- Define and deploy the demo application to the Environment using Radius, which provisions the necessary resources to run the application containers in ACI.
+
+## Prerequisites
+
+- [rad CLI]({{< ref "installation#step-1-install-the-rad-cli" >}})
+- [Bicep VSCode extension]({{< ref "installation#step-2-install-the-vs-code-extension" >}})
+- Radius [installed]({{< ref "/guides/operations/kubernetes/kubernetes-install" >}}) on a [supported Kubernetes cluster]({{< ref "/guides/operations/kubernetes/overview#supported-kubernetes-clusters" >}})
+- An Azure provider configured and registered with your Radius control plane, either through [Service Principal](https://docs.radapp.io/guides/operations/providers/azure-provider/howto-azure-provider-sp/) or [Workload Identity](https://docs.radapp.io/guides/operations/providers/azure-provider/howto-azure-provider-wi/) that have been assigned to the `Contributor` and `Azure Container Instances Contributor` roles on the subscription or resource group where the ACI containers will be deployed
+- A [managed identity](https://learn.microsoft.com/en-us/entra/identity/managed-identities-azure-resources/) is [required]({{< ref "/reference/resource-schema/core-schema/environment-schema#identity" >}}) for ACI deployments, if you choose to utilize a [user-assigned managed identity](https://learn.microsoft.com/en-us/entra/identity/managed-identities-azure-resources/how-manage-user-assigned-managed-identities?pivots=identity-mi-methods-azp) then you need to ensure it is assigned to the `Contributor` and `Azure Container Instances Contributor` roles on the subscription or resource group where the ACI containers will be deployed
+
+## Step 1: Create a Radius Resource Group and Workspace
+
+Since the Radius control plane is hosted on your Kubernetes cluster, you'll need to create a new Radius Resource Group and Workspace so that you may target your application deployments to an ACI Environment. These will then be associated with the ACI Environment that you will configure and create in subsequent steps. 
+
+1. Create a new Radius Resource Group called `aciGroup`:
+   ```bash
+   rad group create aciGroup
+   ```
+
+1. Then, create a new Radius Workspace called `aci-workspace`:
+   ```bash
+   rad workspace create kubernetes aci-workspace
+   ```
+
```

Comment:
We also want to create the environment and switch to the environment similar to what is done for the resource group. This is so that we can update the config.yaml and ensure that the environment is tied to the correct workspace (note that nothing is really deployed when we do this, it's just for config purposes). So: 

`rad env create aci-demo -w aci-workspace` then `rad env switch aci-demo`

Reply:
done